src/net_processing.cpp

```diff
@@ -1758,11 +1756,8 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             }
 
             // Get recent addresses
-            if (pfrom->fOneShot || pfrom->nVersion >= CADDR_TIME_VERSION || connman->GetAddressCount() < 1000)
```

Comment:
`GetAddressCount` is no longer used and can be removed, no? :-)

Reply:
Indeed! Dropped `GetAddressCount` as well.